### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by SurajNarang**
In addition to the past commit messages, I've also improved and added segments to the tutorial video

### CODE REVIEW

File: play/symphony-conductor/index.html [hunk 1]
```diff
@@ -236,6 +237,24 @@ <h3 id="label10">Video Tutorial</h3> <h3 id="label11">Examples</h3>
     <script src="../js/lzma.js"></script>
 
     <script src="js/app.js"></script>
+
+
+    <div id="LoadingGif" style="display:none"> <img src="img/LoadingGif.gif"></div>
+    <div id="LoadingBack" style="display:none"> <img src="img/LoadingBack2.png" width="100%" height="100%"></div>
+
+    <script>
```
Comment (reeswrites) [inline play/symphony-conductor/index.html]: You don't have to put this script inline, you could put it in app.js

File: play/symphony-conductor/js/app.js [hunk 2]
```diff
@@ -1,26 +1,15 @@
 const APP_NAME = "symphony-conductor";
 const hasLocalStorage = checkForLocalStorage();
+var my_lzma = new LZMA("../js/lzma_worker.js");
```
Comment (reeswrites) [inline play/symphony-conductor/js/app.js]: Use ```const``` for this!

File: play/symphony-conductor/js/app.js [hunk 3]
```diff
@@ -316,9 +339,9 @@ function createInstrumentElements() {
 
         // have tooltip show name on hover
         el.setAttribute("data-toggle", "tooltip");
-        el.setAttribute("title", instrument.description + 
-        `<br><a href="${instrument.link}" target="_blank">Click to learn more about this instrument</a>`); 
-        
+        el.setAttribute("title", instrument.description +
```
Comment (reeswrites) [inline play/symphony-conductor/js/app.js]: You shouldn't use concatenation and a template string, just put ```instrument.description``` inside of the template string

File: play/symphony-conductor/index.html [hunk 4]
```diff
@@ -200,16 +201,16 @@ <h3 id="label10">Video Tutorial</h3> <h3 id="label11">Examples</h3>
 
                 <button onclick="myFunction(`myInput2`)" id="exampleButton2">Copy (Complex Level)</button>
 
-                <iframe width="449" height="310" src="https://www.youtube.com/embed/0kg-ZQXIpFY" id="tutorialVideo" >
-            </iframe>
+                <iframe width="449" height="310" src="https://www.youtube.com/embed/fRTmOHbHCJM" id="tutorialVideo">
+                </iframe>
 
                 <script>
                     function myFunction(name) {
```
Comment (reeswrites) [inline play/symphony-conductor/index.html]: This is not a descriptive function name and this could also be in app.js

Comment (SurajNarang) [reply play/symphony-conductor/index.html]: I improved the variable names and put the function into app.js

File: play/symphony-conductor/index.html (hunk 1 above)
Comment (SurajNarang) [reply play/symphony-conductor/index.html]: I transferred this function into app.js

**Pull request merged**